### PR TITLE
Add Rbac chapter

### PR DESCRIPTION
**Pull request by geremyCohen**
https://github.com/aws-samples/eks-workshop/issues/389

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by geremyCohen** [comment]
@brentley @dalbhanj, steps were already verified by @jsunmapr FWIW

File: content/intro_to_rbac/create_iam_user.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+For the sake of simplicity, in this chapter, we will save credentials to a file to make it easy to toggle back and forth between users.  Never do this in production or with credentials that have priveledged access; It is not a security best practice to store credentials on the filesystem.
+{{% /notice %}}
+
+From within the Cloud9 terminal, create a new user called rbac-user, and generate credentials for it:
+
+{{< tabs name="Create EKS Cluster" >}}
+
+{{{< tab name="Workshop at AWS event"  >}}
+
+From within the Cloud9 terminal, run the script given to you by the AWS Workshop Staff.
```
Comment (dalbhanj) [inline content/intro_to_rbac/create_iam_user.md]: I'm not sure why you are saying this...

File: content/intro_to_rbac/create_iam_user.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+{{% notice note %}}
+For the sake of simplicity, in this chapter, we will save credentials to a file to make it easy to toggle back and forth between users.  Never do this in production or with credentials that have priveledged access; It is not a security best practice to store credentials on the filesystem.
+{{% /notice %}}
+
+From within the Cloud9 terminal, create a new user called rbac-user, and generate credentials for it:
+
+{{< tabs name="Create EKS Cluster" >}}
+
+{{{< tab name="Workshop at AWS event"  >}}
```
[the remark this answers is not in the document]
Comment (kornelbicskei) [reply content/intro_to_rbac/create_iam_user.md]: That's a feature not a bug :) EE accounts don't give IAM edit permissions to students so we provide them with pre-generated IAM users. 'rbac-user' in this case.

[the remark this answers is not in the document]
Comment (geremyCohen) [reply content/intro_to_rbac/create_iam_user.md]: @dalbhanj ditto what @kornel said.  For now, for workshop classes, instructors will provide the scripts to run.  In the longer term, we'll tweak the CloudFormation template to auto create this user for each account, and modify the docs accordingly.

File: content/intro_to_rbac/create_iam_user.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+weight: 20
+---
+
+{{% notice note %}}
+For the sake of simplicity, in this chapter, we will save credentials to a file to make it easy to toggle back and forth between users.  Never do this in production or with credentials that have priveledged access; It is not a security best practice to store credentials on the filesystem.
+{{% /notice %}}
+
+From within the Cloud9 terminal, create a new user called rbac-user, and generate credentials for it:
+
+{{< tabs name="Create EKS Cluster" >}}
```
[the remark this answers is not in the document]
Comment (geremyCohen) [reply content/intro_to_rbac/create_iam_user.md]: @dalbhanj busted for copy and paste :)  I modified it to "on your own" (vs at an aws event).  Nice catch.

[the remark this answers is not in the document]
Comment (geremyCohen) [reply content/intro_to_rbac/create_iam_user.md]: done

File: content/intro_to_rbac/install_test_pods.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+```
+kubectl create namespace rbac-test
+kubectl create deploy nginx --image=nginx -n rbac-test
+```
+
+To verify the test pods were properly installed, run:
+
+```
+kubectl get all -nrbac-test
```
[the remark this answers is not in the document]
Comment (geremyCohen) [reply content/intro_to_rbac/install_test_pods.md]: done

File: content/intro_to_rbac/install_test_pods.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+pod/nginx-5c7588df-8mvxx   1/1     Running   0          48s
+
+NAME                    READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/nginx   1/1     1            1           48s
+
+NAME                             DESIRED   CURRENT   READY   AGE
+replicaset.apps/nginx-5c7588df   1         1         1       48s
+
+NAME                          AGE
+mesh.appmesh.k8s.aws/dj-app   9d
```
[the remark this answers is not in the document]
Comment (geremyCohen) [reply content/intro_to_rbac/install_test_pods.md]: done

File: content/intro_to_rbac/create_iam_user.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (geremyCohen) [reply content/intro_to_rbac/create_iam_user.md]: @dalbhanj we're going to modify the CF template to auto-create this user.  In the meantime, due to permissions settings, its not allowed to create a user.

File: content/intro_to_rbac/create_iam_user.md (hunk 1 above)
Comment (geremyCohen) [reply content/intro_to_rbac/create_iam_user.md]: @dalbhanj we're going to modify the CF template to auto-create this user. In the meantime, due to permissions settings, its not allowed to create a user, so we provide the students with a script that has creds for a user pre-created just for the workshop to use instead.

**Pull request merged**